topics/statistics/tutorials/hyperdimensional_computing/tutorial.md

```diff
@@ -0,0 +1,147 @@
+>
+> The tool will end up creating a `summary` file in your history containing a few basic information about the generated classification model and, more importantly, it's accuracy.
+>
+> We suggest you to repeat the same steps for building a classification model on the binary presence/absence profiles in `BIN__ThomasAM__species.csv`. In this case there is no need to use so many levels since the dataset contains only two possible values (i.e., 0 and 1). Thus, the number of levels must be changed to 2.
+>
+>    > <question-title></question-title>
+>    >
+>    > Compare the `summary` files generate by `chopin2` as the result of building a classification model over the `RA__ThomasAM__species.csv` and `BIN__ThomasAM__species.csv` datasets.
+>    > 1. Is there a difference in the accuracy of the two models?
+>    > 2. Does the difference in the number of levels have an impact on the running time and the final accuracy of the models?
+>    {: .question}
+{: .hands_on}
+
+# Feature selection
+
+`chopin2` also implements a feature selection method based on the backward variable elimination strategy. It means that the tool will produce a classification model starting with the whole set of features in the dataset and iteratively remove those features that do not contribute to the accuracy of the model.
+
+Be aware that this specific type of feature selection method could lead to the generation of thousands of classification models in order to determine the best features and discard those ones that do not significantly contribute to a good accuracy. However, despite the huge amount of computational resources required to run the algorithm, it can easily handle datasets with massive amounts of features.
+
+> <hands-on-title>Identify the best features</hands-on-title>
+> **chopin2** {% icon tool %}: Run **chopin2** with the following configuration:
+>    * Select the `BIN__ThomasAM__species.csv` dataset imported into the history during the **Get the data** step;
+>    * Set a dimensionality of 10,000;
+>    * Set 2 levels;
+>    * Set 10 retraining iterations;
+>    * Set the number of folds to 5 to cross-validate the model;
+>    * Finally, enable the feature selection.
+>
+> We are going to focus on the `BIN__ThomasAM__species.csv` dataset only this time since `chopin2` produced a classification model with a better accuracy compared to that of the model built over the `RA__ThomasAM__species.csv` dataset.
+>
+> The tool will end up creating a `summary` file in your history containing a line for each step of the backward variable elimination method. Every line corresponds to a classification model. Please note that the last column reports the features excluded in each step.
+>
+> A `selection` file will also appear in your history with the list of best features selected by `chopin2`.
```

Comment:
A biological interpretation would be nice. What do we  see in the end comparing the stool datasets ...